README.md

```diff
@@ -34,12 +49,6 @@ npm start
 
 4. Open a browser and navigate to http://localhost:3000
```

Comment:
are you sure this port is the one we use? i haven't been able to find it in the code

Reply:
gotcha, thanks :+1: